src/test/scala/com/campudus/tableaux/TableauxTestBase.scala

```diff
@@ -20,19 +21,25 @@ case class TestCustomException(message: String, id: String, statusCode: Int) ext
 trait TableauxTestBase extends TestVerticle {
   val DEFAULT_PORT = 8181
 
+  lazy val config: JsonObject = jsonFromFile("../conf.json")
+  lazy val databaseAddress: String = config.getObject("database", Json.obj()).getString("address", "campudus.asyncdb")
+
+  private def readJsonFile(f: String): String = Source.fromFile(s"$f").getLines().mkString
```

Comment:
Two things here:
1. `Source.fromFile(f).getLines.mkString` - no need for `s"$f"`. `getLines()` is a getter and usually you try to avoid parentheses when using them
2. Why don't you use vertx to read files? :) This could be moved into `asyncBefore()` and done there.